compiler/sigmatch.nim

```diff
@@ -2000,6 +2000,13 @@ proc paramTypesMatchAux(m: var TCandidate, f, a: PType,
           m.baseTypeMatch = true
         else:
           result = userConvMatch(c, m, base(f), a, arg)
+          # bug #4799, varargs accepting subtype relation object
+          if result.isNil and r == isSubtype:
```

Comment:
The check for ``r == isSubtype`` needs to be moved above ``userConvMatch``, I think.

Reply:
Thank you for the tip, I will take a look, after #7600 and #7601